From 93c9db9b739c196b8e7707f29b640251f9b955d9 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@pulkomandy.tk>
Date: Sun, 26 Mar 2017 19:46:53 +0200
Subject: Hack in libnetwork.

I think this could be added to sdl-net config automatically?

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8b09e30..a2f40f6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -130,16 +130,19 @@ if(USE_ANA_NETWORK)
 		${common-external-libs}
 		${ANA_LIBRARIES}
 		${Boost_SYSTEM_LIBRARIES}
+		network
 )
 else(USE_ANA_NETWORK)
 	set(server-external-libs
 		${common-external-libs}
 		${SDLNET_LIBRARY}
 		${Boost_SYSTEM_LIBRARIES}
+		network
 	)
 	set(game-external-libs
 		${game-external-libs}
 		${SDLNET_LIBRARY}
+		network
 	)
 endif(USE_ANA_NETWORK)
 
-- 
2.7.0


From 64b8e91631d62acd90ef738432671b273d43648e Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@pulkomandy.tk>
Date: Mon, 14 Aug 2017 15:50:37 +0200
Subject: Fix undefined reference at runtime.


diff --git a/src/gui/auxiliary/tips.cpp b/src/gui/auxiliary/tips.cpp
index 65d2121..49cff12 100644
--- a/src/gui/auxiliary/tips.cpp
+++ b/src/gui/auxiliary/tips.cpp
@@ -31,6 +31,10 @@ ttip::ttip(const t_string& text,
 {
 }
 
+ttip::~ttip()
+{
+}
+
 namespace tips
 {
 
diff --git a/src/gui/auxiliary/tips.hpp b/src/gui/auxiliary/tips.hpp
index 6b79955..397d1cf 100644
--- a/src/gui/auxiliary/tips.hpp
+++ b/src/gui/auxiliary/tips.hpp
@@ -55,6 +55,8 @@ std::vector<ttip> shuffle(const std::vector<ttip>& tips);
 class ttip
 {
 public:
+	~ttip();
+
 	const t_string& text() const
 	{
 		return text_;
-- 
2.7.0

